README.md

```diff
@@ -43,6 +43,18 @@ On Linux and Mac `dev-env` can be installed with:
 1. Install Nix by running: `bash <(curl https://nixos.org/nix/install)`
 2. Enter `dev-env` by running: `eval "$(dev-env/bin/dade assist)"`
 
+> Note: For **macOS Catalina** users, there are some extra steps required to
+> get `nix` working, as it requires access to a top-level `/nix` directory. The
+> following steps will create a suitable `/nix` folder.
+> 1. Run `sudo bash -c "echo nix >> /etc/synthetic.conf"`
+> 1. Reboot your machine.
+> 1. Run `sudo diskutil apfs addVolume disk1 APFSX Nix -mountpoint /nix`
+> 1. Run `sudo bash -c 'echo "LABEL=Nix /nix apfs rw" >> /etc/fstab'`
+> 1. Reboot your machine.
```

Comment:
Lots of people read Markdown as plaintext. It'd be nice to use the actual sequence numbers rather than all 1s.